### ISSUE TITLE
meteor 1.3 Module '"angular2/router"' resolves to a non-module entity and cannot be imported using this construct.

### ISSUE THREAD

**Issue by vacarsu** [Urigo/angular2-meteor#192]
On a fresh meteor 1.3-rc12 project. I get the following logged in the terminal 
```
Module '"angular2/router"' resolves to a non-module entity and cannot be imported using this construct.
```

also in the web console it logs this
```
Uncaught Error: Cannot find module 'angular2-meteor'
```
package.json
``` javascript
{
  "name": "conflare",
  "private": true,
  "scripts": {
    "start": "meteor run"
  },
  "dependencies": {
    "angular2": "2.0.0-beta.9",
    "angular2-meteor-auto-bootstrap": "git+https://github.com/Urigo/angular2-meteor-auto-bootstrap.git",
    "angular2-meteor-test": "git+https://github.com/Urigo/angular2-meteor.git",
    "es6-shim": "^0.35.0",
    "meteor-node-stubs": "~0.2.0",
    "reflect-metadata": "^0.1.2",
    "rxjs": "^5.0.0-beta.2",
    "zone.js": "^0.6.6"
  }
}
```
packages
```
meteor-base             # Packages every Meteor app needs to have
mobile-experience       # Packages for a great mobile UX
mongo                   # The database Meteor supports right now
reactive-var            # Reactive variable for tracker
jquery                  # Helpful client-side library
tracker                 # Meteor's client-side reactive programming library

standard-minifier-css   # CSS minifier run for production mode
standard-minifier-js    # JS minifier run for production mode
es5-shim                # ECMAScript 5 compatibility for older browsers.
ecmascript              # Enable ECMAScript2015+ syntax in app code

autopublish             # Publish all data to the clients (for prototyping)
insecure                # Allow all DB writes from clients (for prototyping)

angular2-compilers@0.5.1
accounts-password
aldeed:collection2
barbatus:ng2-meteor-accounts
zrbfly:ng2-bootstrap
planettraining:material-design-icons-font
```
Also when I modify .ts files it creates .map and .js file when I save. What is going on?

**Comment by barbatus** [comment]
What is angular2-meteor-test? Use npm install angular2-meteor, npm install angular2-meteor-auto-bootstrap. Also update angular2-compilers package.

**Comment by vacarsu** [comment]
angular2-meteor-test is what gets installed when I use npm install angular2-meteor..

**Comment by vacarsu** [comment]
I deleted the node_modules folder changed angular2-meteor-test in package.json to angular2-meteor, which fixed the first problem. but now the console is logging ```Cannot find module 'angular2/core'```

**Comment by barbatus** [comment]
What if you try this example https://github.com/Urigo/angular2-meteor/tree/master/examples/todos-meteor-1.3 (run `npm install`, then `meteor`)?

**Comment by dohjon** [comment]
I had the same problem. Running: `npm install angular2-meteor`
Installed `"angular2-meteor-test": "git+https://github.com/Urigo/angular2-meteor.git",` for me as well. 
I solved this issue by specifying (anything above 0.5.0 installs `angular2-meteor-test` for me): 
```
  "dependencies": {
    "angular2-meteor": "^0.5.0",
    "angular2-meteor-auto-bootstrap": "^0.5.0"
  } 
```
Even tho I installed `meteor add angular2-compilers@0.5.1` explicitly to newest version 0.5.1 which should fix the peer dependencies, it did not work for me. So I had to install all angular2 peer dependencies manually to get around: `Uncaught Error: Cannot find module 'angular2-meteor' `
```
  "dependencies": {
    "angular2": "^2.0.0-beta.9",
    "angular2-meteor": "^0.5.0",
    "angular2-meteor-auto-bootstrap": "^0.5.0",
    "es6-promise": "^3.0.2",
    "es6-shim": "^0.33.3",
    "reflect-metadata": "^0.1.2",
    "rxjs": "^5.0.0-beta.2",
    "zone.js": "^0.5.15"
  }
```

But now using nested child routers is not working (`OnInit` is not called in nested child routers) which seems to occur due to wrong loading order of scripts (most likely caused by me adding peer dependencies manually?) [https://github.com/angular/angular/issues/4809](https://github.com/angular/angular/issues/4809)
